types/promise.d.ts

```diff
@@ -35,6 +36,7 @@ declare class Database {
     aggregate(name: any, options: any): void;
     table(name: any, factory: any): void;
     loadExtension(...args: any[]): void;
+    maxWriteReplicationIndex(): any;
```

Comment:
Why `any` and not `number | undefined`?

Reply:
This is generated so you have to ask the generator :)